element_animal/injection.py

```diff
@@ -0,0 +1,129 @@
+import importlib
+import inspect
+
+import datajoint as dj
+
+from . import surgery
+
+schema = dj.Schema()
+
+_linking_module = None
+
+
+def activate(
+    injection_schema_name: str,
+    surgery_schema_name: str = None,
+    *,
+    create_schema: bool = True,
+    create_tables: bool = True,
+    linking_module: bool = True,
+):
+    """Activate this schema.
+
+    Args:
+        schema_name (str): schema name on the database server to activate the
+                        `subject` element
+        create_schema (bool): when True (default), create schema in the
+                            database if it does not yet exist.
+        create_tables (bool): when True (default), create tables in the
+                            database if they do not yet exist.
+        linking_module (bool): a module name or a module containing the
+        required dependencies to activate the `subject` element:
+
+    Dependencies:
+    Upstream tables:
+        User: the who conducted a particular surgery/implantation
+    """
+
+    if isinstance(linking_module, str):
+        linking_module = importlib.import_module(linking_module)
+    assert inspect.ismodule(
+        linking_module
+    ), "The argument 'linking_module' must be a module's name or a module"
+
+    global _linking_module
+    _linking_module = linking_module
+
+    surgery.activate(
+        surgery_schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        linking_module=linking_module,
+    )
+    schema.activate(
+        injection_schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        add_objects=linking_module.__dict__,
+    )
+
+
+@schema
+class AAVSerotype(dj.Lookup):
```

Comment:
Let's consider updating the table name to `VirusSerotype`.  In which, we can add `virus_name` as a secondary attribute and remove the `VirusName` table.

Reply:
Updated to `VirusSerotype`.